### DISCUSSION TITLE
For windows platform any popups don't work.

### DISCUSSION THREAD

**Discussion by garawaa** [unoplatform/uno#10760] [Q&A]
I tried some message dialog and file open dialogs but none of these work for me. Some code works other windows platform. what should I do?

**Comment by limeniye** [comment]
<h3>For popup case try it:</h3>

```XML
<Button x:Name="addNewBoard" Content="Add" Height="40" Margin="20,30" CornerRadius="10" Background="#1f1f1f" Foreground="White"
        VerticalAlignment="Bottom" HorizontalAlignment="Stretch">
    <Interactivity:Interaction.Behaviors>
        <Core:EventTriggerBehavior EventName="Click">
            <Core:ChangePropertyAction TargetObject="{Binding ElementName=addApiPopup}" PropertyName="IsOpen" Value="True"/>
        </Core:EventTriggerBehavior>
    </Interactivity:Interaction.Behaviors>
</Button>

<Popup x:Name="addApiPopup" IsLightDismissEnabled="True" PlacementTarget="{Binding ElementName=addNewBoard}" DesiredPlacement="RightEdgeAlignedBottom">
    <Border Height="200" Width="200" CornerRadius="20" Background="#1f1f1f" Padding="20" Spacing="5"/>
</Popup>
```

<h3>For file open dialogs:</h3>

https://platform.uno/docs/articles/features/windows-storage-pickers.html

[iOS sample](https://github.com/unoplatform/uno/blob/22ababc27b7290c15ed4307018f7e4f9bd2bb5e8/doc/articles/features/windows-storage-pickers.md)


If you still have any problems -- please, add information about your code, package and SDK versions.
